inductiva/tests/tasks/test_run_task.py

```diff
@@ -0,0 +1,51 @@
+"""Test file for the run_simulation file."""
+import os
+import json
+
+import pathlib
+
+from inductiva import tasks
+
+LOGGED_TASK_ID = "1234"
+NOT_LOGGED_TASK_ID = "4321"
+
+
+def _id_in_metadata_file(task_id):
+    """Checks if the ID exists in the file."""
+    with open(tasks.TASK_METADATA_FILENAME, "r", encoding="utf-8") as f:
+        json_list = [json.loads(line) for line in f]
+    return task_id in map(lambda x: x["task_id"], json_list)
+
+
+def test_run_simulation_logging(mocker):
+    """Tests if the id of the task was added to the file."""
+
+    # Old logs may cause problems so we remove the file.
+    if os.path.exists(tasks.TASK_METADATA_FILENAME):
```

Comment:
Maybe we should mock the file as well to avoid this. In this case, we. An start from a barebones file and not have problems with what was previously appended.

Reply:
Could you elaborate a bit on mocking the file?